playbooks/coreos-bootstrap.yml

```diff
@@ -3,6 +3,7 @@
   gather_facts: False
   environment: "{{ proxy_env }}"
   roles:
+    - coreos_bootstrap
```

Comment:
no, as the bootstrap tasks checks if its already been run, and skips if so but will add a toggle in case